src/test/spec/json/unified-runner-examples/poc-crud.json

```diff
@@ -0,0 +1,446 @@
+{
```

Comment:
do we still need this example files? or are they now included in their respective spec folders (e.g. crud)?

Reply:
Yep, these are the only tests that have been written so far for what we have implemented in the driver, so until the rest of the tests are ported we need to keep these around to ensure the runner is working.